odoc.opam

```diff
@@ -24,6 +24,10 @@ Odoc is a documentation generator for OCaml. It reads doc comments,
 delimited with `(** ... *)`, and outputs HTML. 
 """
 
+pin-depends: [
+  ["odoc-parser.dev" "git+https://github.com/ocaml-doc/odoc-parser.git#f98cfe3"]
```

Comment:
Yes, a release of `odoc-parser` is needed. But we also shouldn't release `odoc-parser` without releasing `odoc`...

Also, since https://github.com/ocaml-doc/odoc-parser/pull/16 has also been merged but not its odoc counterpart #950, this `pin-depends` don't work anyway...

Reply:
The pin-depends has been updated, and "support" added to this PR. So this is now resolved.